### PR TITLE
Update java lib/wrapper

### PR DESCRIPTION
**Pull request by wurstbrot**
Bash wrapper is not supported anymore due to java library/image.

### CODE REVIEW

File: WRAPPERS.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 | -----------------------------| -------|   ---- | ------------------------|
 | Specific python wrapper      | Yes    | Yes    | working (2021-01-21)    |
 | Openapi python wrapper       | No     | Yes    | proof of concept only   |
-| Bash wrapper                 | No     | Yes    | might be outdated       |
-| Java library                 | No     | Yes    | working (2021-04-13)    |
+| Java library                 | No     | Yes    | working (2021-08-30)    |
 | Image using the Java library | No     | Yes    | out of date             |
```
Comment (StefanFl) [inline WRAPPERS.md:6]: Is this image really still out of date, when it depends on the Java library which is working?

**Review by StefanFl — CHANGES_REQUESTED** [review]
It might be better to change the PR so it will be merged in the `dev` branch. There are already changes for `WRAPPERS.md` there.

**Comment by wurstbrot** [comment]
Close in favor of https://github.com/DefectDojo/django-DefectDojo/compare/dev...wurstbrot:patch-4

**Comment by StefanFl** [comment]
Will you create a new PR?

**Pull request closed without merging**